pykeepass/pykeepass.py

```diff
@@ -209,6 +215,12 @@ def transformed_key(self):
         and passed to `open` for faster database opening"""
         return self.kdbx.body.transformed_key
 
+    @property
+    def database_salt(self):
```

Comment:
Needs a test, if possible a small line explaining what it is too.

Reply:
Ok, will add one.

Reply:
Added.